integration_tests/src/main/python/avro_test.py

```diff
@@ -140,13 +140,24 @@ def test_avro_read_with_corrupt_files(spark_tmp_path, reader_type, v1_enabled_li
 
 @pytest.mark.parametrize('v1_enabled_list', ["avro", ""], ids=["v1", "v2"])
 @pytest.mark.parametrize('reader_type', rapids_reader_types)
-def test_read_count(spark_tmp_path, v1_enabled_list, reader_type):
+@pytest.mark.parametrize('batch_size_rows', [10, 2147483647])
+def test_avro_read_count(spark_tmp_path, v1_enabled_list, reader_type, batch_size_rows):
```

Comment:
`test_read_count` is enough, because it is in the file "avro_test.py" which already indicates these tests are for avro.